packages/clerk-js/src/core/resources/SignUp.ts

```diff
@@ -487,11 +488,34 @@ class SignUpFuture implements SignUpFutureResource {
     return this.resource.unverifiedFields;
   }
 
+  private async getCaptchaToken(): Promise<{ captchaToken?: string; captchaWidgetType?: CaptchaWidgetType }> {
+    const captchaChallenge = new CaptchaChallenge(SignUp.clerk);
+    const response = await captchaChallenge.managedOrInvisible({ action: 'signup' });
+    if (!response) {
+      throw new Error('Captcha challenge failed');
```

Comment:
Should this use `ClerkRuntimeError`? Or not because it's only internal?

Reply:
this is me kicking that can down the road until I can do an error pass across the entire implementation 😅